dwave_networkx/algorithms/tsp.py

```diff
@@ -148,6 +148,13 @@ def traveling_salesperson_qubo(G, lagrange=2, weight='weight'):
     """
     N = G.number_of_nodes()
 
+    if lagrange is None:
+        # if no lagrange parameter provided, set to 'average' tour length
+        if G.number_of_edges()>0:
+            lagrange = G.size(weight=weight)*G.number_of_nodes()/G.number_of_edges()
```

Comment:
Could you add few words in the comment about why it makes sense to set this value for the parameter?

Reply:
Added.